<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text-Speech-Translator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>181</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Type What You Want To Say</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_input">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>521</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="speech_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>350</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Speak</string>
    </property>
   </widget>
   <widget class="QComboBox" name="LangCombo">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>78</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>English</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spanish</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>French</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Russian</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dutch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Korean</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Japanese</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chinese</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>310</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>What Language?</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
